models/prices/ethereum/prices_ethereum_tokens.sql

```diff
@@ -1621,7 +1621,9 @@ FROM
     ('gnx-genaro-network', 'ethereum', 'GNX', 0x6ec8a24cabdc339a06a172f8223ea557055adaa5, 9),
     ('pndc-pond-coin', 'ethereum', 'PNDC', 0x423f4e6138e475d85cf7ea071ac92097ed631eea, 18),
     ('crvusd-curvefi-usd-stablecoin', 'ethereum', 'CRVUSD', 0xf939e0a03fb07f59a73314e73794be0e57ac1b4e, 18),
-    ('mav-maverick-token', 'ethereum', 'MAV', 0x7448c7456a97769f6cd04f1e83a4a23ccdc46abd, 18)
+    ('mav-maverick-token', 'ethereum', 'MAV', 0x7448c7456a97769f6cd04f1e83a4a23ccdc46abd, 18),
+    ('frxeth-frax-ether', 'ethereum', 'frxETH', 0x5e8422345238f34275888049021821e8e08caa1f, 18),
+    ('sdt-stakedao-token', 'ethereum', 'SDT', 0x73968b9a57c6e53d41345fd57a6e6ae27d6cdb2f, 18)
```

Comment:
@0xpibbles it looks like this is an invalid id:
https://api.coinpaprika.com/v1/coins/sdt-stakedao-token